shopify-order-to-sap-order/main.bal

```diff
@@ -0,0 +1,108 @@
+import ballerina/http;
+import ballerina/log;
+
+const SAP_REQUEST_PATH = "/purchaseorder/0001/PurchaseOrder";
+const SAP_URL = "https://my401785.s4hana.cloud.sap/sap/opu/odata4/sap/api_purchaseorder_2/srvd_a2x/sap/";
+const HTTP_STATUS_OK = 200;
+const HTTP_STATUS_CREATED = 201;
+const HEADER_KEY_CSRF_TOKEN = "x-csrf-token";
+const HEADER_FETCH_VALUE = "fetch";
+
+map<string> supplierMap = {"7336631533890": "17300096"};
```

Comment:
```suggestion
final map<string> supplierMap = {"7336631533890": "17300096"};
```

Reply:
You can make this to intersection with readonly. That way you can make the resources isolated.

Reply:
Addressed with 5a7741e